openapi/openapi.yaml

```
@@ -2,6 +2,121 @@ openapi: 3.0.3
 
 info:
   version: 1.0.0
-  title: Inventario de Recambios
+  title: 🚗[AOS 2023] Inventario de taller de coches 🚗
```

Me parece bien poner emojis, pero creo que la palabra clave para este servicio es "Recambios".
¿No verías mejor especificarlo tal que así?
```suggestion
  title: 🚗[AOS 2023] Inventario de Recambios del taller 🚗
```
En este caso no he puesto taller de coches porque se puede asumir con los emojis.
También he puesto en mayúscula la inicial de Recambios para resaltarlo.